packages/cli/project.json

```diff
@@ -19,6 +19,14 @@
       "command": "node tools/scripts/publish.mjs cli {args.ver} {args.tag}",
       "dependsOn": ["build"]
     },
+    "version": {
+      "postTargets": ["cli:github"]
+    },
+    "github": {
+      "options": {
+        "notesFile": "./packages/cli/CHANGELOG.md"
+      }
+    },
```

Comment:
What about `ngx-deploy-npm`? Is that another PR? Just checking.

Reply:
yes